packages/fluent-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -48,12 +44,6 @@ export default function CheckboxesWidget<
     }
   };
 
-  const _onBlur = ({ target: { value } }: FocusEvent<HTMLButtonElement>) =>
-    onBlur(id, enumOptionsValueForIndex<S>(value, enumOptions, emptyValue));
-
-  const _onFocus = ({ target: { value } }: FocusEvent<HTMLButtonElement>) =>
-    onFocus(id, enumOptionsValueForIndex<S>(value, enumOptions, emptyValue));
-
```

Comment:
Same questions about onBlur/onFocus

Reply:
Same as above. Updated in my latest commit.